app/layouts/DashboardLayout/BalanceIndicator.view/BalanceIndicator.d.ts

```diff
@@ -0,0 +1,4 @@
+export interface BalanceIndicatorProps {
+  balance: string | '';
```

Comment:
Why the `| ''` part? It's still a string.

Reply:
Someone else wrote this particular type, I just exported the interface. Wasn't sure (and still not) if there's a scenario where removing this would cause a user not to be able to see there balance.

Reply:
Should be covered in a test, let's just remove the | '' part and move forward :)

Reply:
👍🏾

Reply:
Agree: remove the ''